pytest_html/plugin.py

```diff
@@ -86,7 +88,7 @@ def data_uri(content, mime_type='text/plain', charset='utf-8'):
 
 class HTMLReport(object):
 
-    def __init__(self, logfile):
+    def __init__(self, logfile, **kwargs):
```

Comment:
Let's explicitly list the keyword arguments - it provides better documentation and usability of the class.